spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -184,6 +184,28 @@
       and_the_steps_can_be_checked_for_broken_links
     end
 
+    scenario "User schedules publishing with a public change note" do
+      given_there_is_a_draft_step_by_step_page_with_secondary_content_and_navigation_rules
+      and_I_visit_the_scheduling_page
+      then_I_should_see_a_publish_form_with_changenotes
+      and_I_fill_in_the_changenote_form
+      and_when_I_click_button "Continue"
+      then_inputs_should_have_tomorrows_date
+      and_I_fill_in_the_scheduling_form
+      when_I_submit_the_form
+      then_I_should_see "has been scheduled to publish"
+      and_the_step_by_step_should_have_the_status "Scheduled"
+      and_there_should_be_a_change_note "Scheduled by Test author for publishing at 10:26am on 20 April 2030 with change note: We made some changes"
```

Comment:
I think everything below this line can be removed (and put in a less complex scenario if it hasn't been covered yet) as this scenario is very long.

Reply:
Good point, I've removed those lines now